core/templates/pages/contributor-dashboard-page/services/contribution-and-review.service.ts

```diff
@@ -119,6 +122,20 @@ angular.module('oppia').factory('ContributionAndReviewService', [
         }).then(function() {
           onSuccess(suggestionId);
         }, () => onFailure && onFailure(suggestionId));
+      },
+      updateTranslation: function(
```

Comment:
`updateTranslationSuggestion` to be more specific.

Reply:
Done